MonoDevelop.MSBuild.Editor/Completion/MSBuildCompletionSource.cs

```diff
@@ -24,11 +18,24 @@
 using MonoDevelop.MSBuild.SdkResolution;
 using MonoDevelop.Xml.Dom;
 using MonoDevelop.Xml.Editor.Completion;
+using static MonoDevelop.MSBuild.Language.ExpressionCompletion;
 using MonoDevelop.Xml.Parser;
 
+using Newtonsoft.Json.Linq;
+
 using ProjectFileTools.NuGetSearch.Feeds;
 
-using static MonoDevelop.MSBuild.Language.ExpressionCompletion;
+using System;
```

Comment:
Per the .editorconfig, System comes before other usings https://github.com/mhutch/MonoDevelop.MSBuildEditor/blob/master/.editorconfig#L42

If you use "Remove and Sort Usings" it will respect this setting.